recipe_scrapers/gourmettraveller.py

```diff
@@ -0,0 +1,78 @@
+# mypy: allow-untyped-defs
+
+import re
+
+from ._abstract import AbstractScraper
+from ._grouping_utils import group_ingredients
+from ._utils import normalize_string
+
+
+class GourmetTraveller(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "gourmettraveller.com.au"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        recipe_category_span = self.soup.find(
+            "span", {"class": "related-tags__label"}, string=re.compile("Recipe Course")
+        )
+        if not recipe_category_span:
+            return None
+        value = recipe_category_span.find_next_sibling("span")
+        return normalize_string(value.text)
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def prep_time(self):
+        return self.schema.prep_time()
+
+    def cook_time(self):
+        return self.schema.cook_time()
+
+    def yields(self):
+        return self.schema.yields()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        group_heading_divs = self.soup.find_all(
+            "div", {"class": "recipe-ingredients__title"}
+        )
+        group_headings = [
+            normalize_string(group_heading_div.text)
+            for group_heading_div in group_heading_divs
+        ]
+
+        # The group headings are also included in the ingredients in the schema
+        # Remove the group headings from the ingredients
+        ingredients = filter(
+            lambda ingredient: ingredient not in group_headings,
+            self.schema.ingredients(),
+        )
+
+        return list(ingredients)
```

Comment:
Optional: I think it'd be possible to translate this into a list comprehension:

```suggestion
        return [
            ingredient
            for ingredient in self.schema.ingredients()
            if ingredient not in group_headings
        ]
```

I'm also trying to think of scenarios where somehow a group heading could also be an ingredient name... is that worth worrying about?

Reply:
I also had a think about this - I couldn't think of a scenario where this would happen, and even if the ingredient name was the same, the ingredients are pretty much always paired with a quantity, and I can't see that being the case for a group heading.

There are two options for change I could think of if you would like to assume that there could somehow be a case where an ingredient matches a group heading:
- Generate the whole list from the HTML
  - Pro: We are already using the HTML to get the group headings
  - Con: I prefer using the data in the schema that the site gives us over what we can scrape manually as it is less brittle
- Augment the existing logic above to only remove one instance of each group heading
  - Pro: If there was an ingredient the same as a group heading, one of the two would still be included somewhere
  - Con: It could eliminate the wrong one (although since this list doesn't have to be ordered, this probably isn't a massive issue, unless `group_ingredients` cares?)

I would probably lean towards the second option, but I would be keen to hear your input @jayaddison